src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -17,34 +15,67 @@
 
 	@Autowired
 	private EvalRepository dao;
+	
+
+//CREATE----------------------------
 
+//RETRIEVE--------------------------
 	@Override
 	public Eval getEvalById(int id) {
-		return dao.findOne(id);
-
+		Eval eval = dao.findOne(id);
+		return eval;
 	}
 	
 	@Override
-	public List<Eval> getEvalsByBatch(int id){
-		
-		return dao.findAllByBatchId(id);
+	public Page<Eval> getEvalsByBatch(Pageable pageable, int id, String evalParam){
+		Page<Eval> evals;
+		if (evalParam.equalsIgnoreCase("both")) {
+			evals = dao.findAllByBatchId(pageable, id);
+			System.out.println(evals);
+		} else {
+			evals = dao.findAllByBatchIdAndEvalTypeDescriptionIgnoreCase(pageable, id, evalParam);
+			System.out.println(evals);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)